packages/documentation/src/stories/components/forms/form-footer/form-footer.stories.ts

```diff
@@ -0,0 +1,117 @@
+import { Args, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { MetaComponent } from '@root/types';
+
+export const FooterArgs = {
+  showPrimaryButton: true,
+  primaryButtonText: 'Send',
+  showSecondaryButton: true,
+  secondaryButtonText: 'Cancel',
+  showTertiaryButton: true,
+  tertiaryButtonText: 'Back',
+};
+
+const meta: MetaComponent = {
+  id: 'f2eddf67-2c3c-40c4-bfec-df49bd028001',
+  title: 'Components/Forms/Form Footer',
+  tags: ['package:HTML'],
+  render: render,
+  parameters: {
+    badges: [],
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=1498-28215',
+    },
+  },
+  args: FooterArgs,
+  argTypes: {
+    showPrimaryButton: {
+      name: 'Show primary button',
+      description: 'Show or hide the primary button (last one on the right)',
+      control: { type: 'boolean' },
+      table: {
+        category: 'Primary button',
+      },
+    },
+    primaryButtonText: {
+      name: 'Primary button text',
+      description: 'Text to display on the primary button',
+      control: { type: 'text' },
```

Comment:
Can you hide the controls for button X when showXButton is false, like so:
```suggestion
      control: { type: 'text' },
      if: {
        arg: 'showPrimaryButton',
      },
```